### PR TITLE
Issue 50356: Remove unused UI that contains usages of javascript: URLs

### PR DESCRIPTION
**Pull request by labkey-susanh**
#### Rationale
Many parts of the workflow UI were developed for demonstration purposes only, with the actual usages being through APIs. 
Rather than maintain this UI that has some problematic bits, we'll simply remove it.

#### Related Pull Requests
* 

#### Changes
* Remove `StartProcessFormAction` in favor of `StartProcessForm`
* Remove support for task completion with parameters in the UI.

### CODE REVIEW

File: src/org/labkey/workflow/view/WorkflowViewBase.java [hunk 1]
```diff
@@ -103,51 +103,6 @@ private Object getParameterValue(String key, Object value)
         return value;
     }
 
-    public String actionForm(String name, String formName, String formAction, Map<String, TaskFormField> fields, @Nullable Map<String, String> hiddenFields)
-    {
-        StringBuilder builder = new StringBuilder();
-        builder.append("<strong>").append(h(name)).append("</strong>");
-        builder.append("<br><br>\n");
-        builder.append("<form name=\"").append(h(formName)).append("\" action=\"").append(h(formAction)).append("\">\n");
-        for (Map.Entry<String, TaskFormField> field : fields.entrySet())
```
Comment (labkey-nicka) [inline src/org/labkey/workflow/view/WorkflowViewBase.java:112]: nit: Unused import `TaskFormField`.

File: src/org/labkey/workflow/view/workflowProcessStart.jsp [hunk 2]
```diff
@@ -1,81 +0,0 @@
...
-<%@ page import="java.util.HashMap" %>
-<%@ page import="java.util.Map" %>
-<%@ page extends="org.labkey.workflow.view.WorkflowViewBase" %>
-<%@ taglib prefix="labkey" uri="http://www.labkey.org/taglib" %>
-<%!
-    @Override
-    public void addClientDependencies(ClientDependencies dependencies)
-    {
-        dependencies.add("Ext4");
-        dependencies.add("workflow/view/startProcess.js");
```
Comment (labkey-nicka) [inline src/org/labkey/workflow/view/workflowProcessStart.jsp:34]: As far as I can tell this is the only usage of `workflow/view/startProcess.js`. Consider removing that script as well.

File: src/org/labkey/workflow/view/workflowTask.jsp [hunk 3]
```diff
@@ -146,12 +146,9 @@
...
                 }
             }
-            else
-            {
+            else {
 %>
-<%= unsafe(actionForm(bean.getName(), bean.getTaskDefinitionKey(),
-        "javascript:completeWorkflowTask(" + q(bean.getId()) + "," + qh(bean.getTaskDefinitionKey()) + ", ['" + StringUtils.join(fields.keySet(), "', '") + "']," + q(bean.getProcessInstanceId()) + ", " + qh(bean.getProcessDefinitionKey(getContainer())) + ")", fields, null)) %>
-<br>
+                <div>Unsupported in UI</div>
```
Comment (labkey-nicka) [inline src/org/labkey/workflow/view/workflowTask.jsp:150]: nit: Unused import `StringUtils`.

**Review by labkey-nicka — APPROVED** [review]

File: src/org/labkey/workflow/view/WorkflowViewBase.java (hunk 1 above)
Comment (labkey-susanh) [reply src/org/labkey/workflow/view/WorkflowViewBase.java:112]: Done

File: src/org/labkey/workflow/view/workflowProcessStart.jsp (hunk 2 above)
Comment (labkey-susanh) [reply src/org/labkey/workflow/view/workflowProcessStart.jsp:34]: Done.

File: src/org/labkey/workflow/view/workflowTask.jsp (hunk 3 above)
Comment (labkey-susanh) [reply src/org/labkey/workflow/view/workflowTask.jsp:150]: Done.

**Pull request merged**